HappyAnding/HappyAnding/Views/ExploreShortcutViews/ExploreShortcutView/ExploreShortcutView.swift

```diff
@@ -75,6 +106,98 @@ struct ExploreShortcutView: View {
+                
+                Spacer()
+                
+                MoreCaptionTextView(text: TextLiteral.more)
+                    .navigationLinkRouter(data: NavigationListShortcutType(sectionType: sectionType,
+                                                                           shortcuts: shortcuts,
+                                                                           navigationParentView: .shortcuts))
+            }
+            .padding(.horizontal, 16)
+            .padding(.bottom, 12)
+            
+            ForEach(Array(shortcuts.enumerated()), id:\.offset) { index, shortcut in
+                if index < 3 {
+                    let data = NavigationReadShortcutType(shortcutID:shortcut.id,
+                                                          navigationParentView: .shortcuts)
+                    
+                    ShortcutCell(shortcut: shortcut,
+                                 rankNumber: index + 1,
+                                 navigationParentView: .shortcuts)
+                    .navigationLinkRouter(data: data)
+                }
+            }
+            .background(Color.shortcutsZipBackground)
+        }
+    }
+    
+    @ViewBuilder
+    private func categoryCardView(with category: Category) -> some View {
+        
+        VStack(spacing: 0) {
+            HStack {
+
+                SubtitleTextView(text: category.translateName())
+
+                Spacer()
+
+                MoreCaptionTextView(text: TextLiteral.more)
+                    .navigationLinkRouter(data: NavigationListCategoryShortcutType(shortcuts: [],
+                                                                                   categoryName: category,
+                                                                                   navigationParentView: .shortcuts))
+            }
+            .padding(.horizontal, 16)
+            .padding(.bottom, 12)
+
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack {
+                    ForEach(Array((shortcutsZipViewModel.shortcutsInCategory[randomCategories[0].index].enumerated())), id: \.offset) { index, shortcut in
+                        if index < 7 {
+                            let data = NavigationReadShortcutType(
+                                shortcutID: shortcut.id,
+                                navigationParentView: .shortcuts)
+
+                            ShortcutCardCell(
+                                categoryShortcutIcon: shortcut.sfSymbol,
+                                categoryShortcutName: shortcut.title,
+                                categoryShortcutColor: shortcut.color)
+                            .navigationLinkRouter(data: data)
+                        }
+                    }
+                }
+                .padding(.horizontal, 16)
+            }
+        }
+    }
+    
+    @ViewBuilder
+    func categoryCellView(with categoryName: String) -> some View {
```

Comment:
이 함수만 internal인 이유가 있을까요?
```suggestion
    private func categoryCellView(with categoryName: String) -> some View {
```

Reply:
아뇨 제가 빼먹었네요 🥲